tic_tac_toe/game.py

```diff
@@ -24,18 +43,35 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    # if x in position
+    result = False
+    
+    if type(position) == tuple and len(position) == 2:
+        if len([x for x in position if x >= 0 and x <= 2 and type(x) == int]) == 2:
+            result = True
+
+    return result
+    
+    '''
+    if (position[0] < 0 or position[0] > 2 and position[1] < 0 or position[1] >2):
+        result = False
+    return result
+    '''
 
 def _board_is_full(board):
     """
     Returns True if all positions in given board are occupied.
 
     :param board: Game board.
     """
-    pass
-
-
+    result = True
+    
+    if (any('-' in sublist for sublist in board)):
```

Comment:
This was the most understandable way I have seen this written. Excellent clarity of thought here.